torchdynamo/convert_frame.py

```diff
@@ -61,16 +61,26 @@ def fx_forward_from_src_skip_result(*args, **kwargs):
 
 def wrap_compiler_fn(compiler_fn):
     """Expand backend strings to functions"""
-    if compiler_fn == "inductor":
+    if isinstance(compiler_fn, WrapperBackend):
+        # currently, compiler_fn's backend is not callable
+        # if it use default BACKENDS
+        # set compiler_fn.backend as the callable function
+        wrapper_compiler_fn = compiler_fn    
+    else:
+        # convert to WrapperBackend class
+        wrapper_compiler_fn = WrapperBackend(compiler_fn)
```

Comment:
I'd suggest only adding `WrapperBackend` in the case of `verify_correctness=True`, and leave the `verify_correctness=False` behavior the same.  That might make things a bit simpler here.  You can add the wrapper after we decode string backends.